### PR TITLE
fix error on no ability

### PR DESCRIPTION
**Pull request by brycerobinsn**
image

I think my last PR didn't have your fix for the no ability crash :)

### CODE REVIEW

File: src/features/bravery/components/RandomItems.jsx [hunk 1]
```diff
@@ -14,12 +14,20 @@ export function RandomItems() {
...
+  if (ability) {
+    abilityTooltipLabel = <TooltipCard header={ability.name} body={ability.description} />;
+    abilityAvatarSrc = `${SPELL_IMAGE_BASE_URL}${ability?.image?.full}`;
+  }
+
   return (
     <Flex marginTop={8} gap={2} justifyContent="space-between">
       <Box>
-        <Tooltip label={<TooltipCard header={ability.name} body={ability.description} /> || ABILITY_ARRAY[state.selectedRandomAbilityIndex]}>
+        <Tooltip label={abilityTooltipLabel} >
```
Comment (cannedyeti) [inline src/features/bravery/components/RandomItems.jsx]:
```suggestion
        <Tooltip label={<TooltipCard header={ability?.name} body={ability.description} /> || ABILITY_ARRAY[state.selectedRandomAbilityIndex]}} >
```

can just do this xd

File: src/features/bravery/components/RandomItems.jsx [hunk 2]
```diff
@@ -14,12 +14,20 @@ export function RandomItems() {
...
           <Avatar
-            src={`${SPELL_IMAGE_BASE_URL}${ability?.image?.full}`}
+            src={abilityAvatarSrc}
```
Comment (cannedyeti) [inline src/features/bravery/components/RandomItems.jsx]:
```suggestion
            src={ability?.name ? `${SPELL_IMAGE_BASE_URL}${ability?.image?.full}` : <WarningTwoIcon />}
```

Rather just do ternary here. Cleaner than random ifs

File: src/features/bravery/components/RandomItems.jsx [hunk 3]
```diff
@@ -14,12 +14,20 @@ export function RandomItems() {
   const ability =
     state.selectedRandomChampion?.spells?.[state.selectedRandomAbilityIndex];
 
+
+  let abilityTooltipLabel = <TooltipCard header={`Start: ${ABILITY_ARRAY[state.selectedRandomAbilityIndex]}`} body="Error Loading Ability"/>
+  let abilityAvatarSrc = <WarningTwoIcon />;
```
Comment (cannedyeti) [inline src/features/bravery/components/RandomItems.jsx]: good to use `const` over `let` if you're never altering the variable. But I don't think we need these after changes are made anyway

**Review by cannedyeti — CHANGES_REQUESTED** [review]

File: src/features/bravery/components/RandomItems.jsx (hunk 1 above)
Comment (brycerobinsn) [reply src/features/bravery/components/RandomItems.jsx]: this was still popping an error. Switched to ternary

**Review by cannedyeti — APPROVED** [review]

**Pull request merged**